Makefile

```diff
@@ -86,10 +86,18 @@ endif
 ifeq ($(TRACE), 1)
   UFTRACE_CFLAGS += -pg -fno-omit-frame-pointer
   DEMANGLER_CFLAGS += -pg -fno-omit-frame-pointer
+  SYMBOLS_CFLAGS += -pg -fno-omit-frame-pointer
   TRACEEVENT_CFLAGS += -pg -fno-omit-frame-pointer
   # cannot add -pg to LIB_CFLAGS because mcount() is not reentrant
 endif
 
+ifeq ($(GCOV), 1)
```

Comment:
Instead of using name of a specific tool, what about a generic name like 'COVERAGE' (provided that it won't called frequently)?

Reply:
Okay. I will change it to `COVERAGE`.